code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm

```diff
@@ -32,6 +32,7 @@
 	sawn = /obj/item/gun/projectile/boltgun/obrez
 	var/bolt_open = 0
 	var/item_suffix = ""
+	wield_delay = 8 //deciseconds
```

Comment:
its sawed off , why 8 deciseconds ? it should be only 4 or 3

Reply:
done